packages/@tinacms/fields/src/plugins/TagsFieldPlugin.tsx

```diff
@@ -26,6 +26,7 @@ import { CloseIcon } from '@tinacms/icons'
 export const TagsField = wrapFieldsWithMeta<
   { placeholder: string },
   InputProps
+  // @ts-ignore
```

Comment:
This ignores an issue with `tinaForm` not being one of the standard set of `props` that a `Field` can accept.  I'm not sure I've seen `tinaForm` used before; I assume `form` is the `final-form` instance and `tinaForm` is the `useForm()` instance?